_plugins/helm.rb

```diff
@@ -27,6 +27,12 @@ def render(context)
 
       # Load the versions.yml file so it can be rewritten in a standard helm format.
       versionFile = YAML::load_file('_data/versions.yml')
+      if not versionFile.key?(version)
```

Comment:
I think this looks good - just want to confirm my understanding that this will leave empty/broken links in the rendered site, so PR builds will fail if they are missing a version.

Reply:
Not quite. The ruby plugin will return an empty string, so the manifest files will exist on the site but will be blank.